Userland/Libraries/LibFileSystemAccessClient/Client.cpp

```diff
@@ -51,7 +54,10 @@ Result Client::request_file(i32 parent_window_id, String const& path, Core::Open
         GUI::WindowServerConnection::the().async_remove_window_stealing_for_client(child_window_server_client_id, parent_window_id);
     });
 
-    async_request_file(parent_window_server_client_id, parent_window_id, path, mode);
+    if (path.starts_with('/'))
+        async_request_file(parent_window_server_client_id, parent_window_id, path, mode);
+    else
+        async_request_file(parent_window_server_client_id, parent_window_id, String::formatted("{}/{}", getcwd(nullptr, 0), path), mode);
```

Comment:
Same here, please use `LexicalPath::join()` - also, do we have a LibCore API for current directory? If so, please prefer that

Reply:
There's `LibCore::File::current_working_directory()`, but that's just a wrapper around this and makes a temporary string and I was just trying to avoid an unnecessary allocation. I'll use it if we prefer using LibCore then I'll change it.